public/package-lock.json

```diff
@@ -4,6 +4,16 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "JSONStream": {
```

Comment:
why did all the dependencies change for this PR?

Reply:
The package-lock.json file also doesn't do anything since we don't use node_modules at all. Vendored dependencies are like 50% listed in the package.json file, but that's mostly used for bookkeeping

Reply:
If the evergreen team are ok with it changing then I have no objection.

Reply:
This happened due to running npm install --> I will go ahead and remove this file since it is superfluous anyhow.